flowmachine/flowmachine/core/server/query_schemas/custom_fields.py

```diff
@@ -46,6 +46,60 @@ def __init__(self, required=False, validate=None, **kwargs):
         )
 
 
+class TotalBy(fields.String):
+    """
+    A string representing a period type, e.g. "day"
+    """
+
+    def __init__(self, required=False, **kwargs):
+        # if validate is not None:
```

Comment:
Can we remove these lines, or at least add an explanation for them being commented out?